jac-uk/qt
# Feat/qt 84 improvements to qt admin dashboard

## What's included?
Closes #84 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Go to the preview link: https://jac-qualifying-tests-admin-develop--feat-84-em6fpqsm.web.app/folder/1WkTLw87DWPwT1v6161q/qualifying-tests/h0CEsyNZZG6LCBuJbwRP/
- Check the content of tooltips, it should provide explanation for different numbers.
<img width="654" alt="Screenshot 2024-10-03 at 09 28 59" src="https://github.com/user-attachments/assets/88dc0b34-a225-462c-baf7-77c07ca85304">
- Click Close & Score, it should recalculate the number of participants.
<img width="1141" alt="Screenshot 2024-10-03 at 12 46 26" src="https://github.com/user-attachments/assets/633b737e-5597-4462-8fec-e6296e47af93">

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Include screen grabs, video demo, notes etc.

## Related permissions
Have permissions been considered for this functionality?
- No permission changes required
- Permissions have been added / updated. Details:

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
